storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11106,6 +11107,81 @@ static void show_myrocks_vars(THD *thd, SHOW_VAR *var, char *buff) {
   var->value = reinterpret_cast<char *>(&myrocks_status_variables);
 }
 
+static ulonglong io_stall_prop_value(std::map<std::string, std::string> *props,
```

Comment:
1st arg: const ref, 2nd arg: rvalue ref?

Reply:
Sure. Not sure how to make a **const**, though, as I use iterator inside.

Reply:
Ok, I think I've figured out how to do that in my latest revision. Thanks.